### PR TITLE
Adding more info for Clients (size client list), and Week size

### PR DESCRIPTION
**Pull request by sakrow**
I add more info for all clients:
- The 10 heavier clients
- Daily backup size

### CODE REVIEW

**Comment by dfranco** [comment]
Hi,

Thanks for the pull request :)
I'll have a look at your code and get back to you soon ;)

Regards

**Comment by feldsam** [comment]
What about this? @sakrow should update his branch with actual master and resolve conflict, so it can be mergeable.

File: application/view/index.tpl [hunk 1]
```diff
@@ -1,90 +1,445 @@
...
+              </tr>
+            {/foreach}
+          </table>
+        </div> <!-- <div class="panel-body"> -->
+      </div> <!-- <div class="panel panel-default"> -->
+    </div> <!-- end class="col-..."-->
+
+  </div> <!-- end <div class="row"> -->
+</div> <!-- end <div class="container"> -->
+=======
```
Comment (feldsam) [inline application/view/index.tpl]: Some left over merging conflict?

File: config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# config valid only for current version of Capistrano
```
Comment (feldsam) [inline config/deploy.rb]: what about this ruby things???

File: core/vendor/bootstrap/css/bootstrap-theme.css [hunk 3]
```diff
@@ -0,0 +1,587 @@
+/*!
```
Comment (feldsam) [inline core/vendor/bootstrap/css/bootstrap-theme.css]: why you add new theme? This pull request is only about adding more info....

File: core/vendor/bootstrap/js/npm.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
```
Comment (feldsam) [inline core/vendor/bootstrap/js/npm.js]: autogenerated file, why is tracked and no in gitignore? also, do we need new boostrap?

**Comment by sakrow** [comment]
Sorry. This "merge" should not be here.
I want to update my branch "master" not "pull request"

**Comment by feldsam** [comment]
Hi, you should all time base your pull requests on separate branch.

**Comment by sakrow** [comment]
Yes, sorry, noob error.

**Comment by dfranco** [comment]
Just for your information, I've added some information for developers in the documentation
http://bacula-web.org/Development.html
Please follow these rules and get back to me if I miss something else.
Regards

**Pull request closed without merging**